README.md

```diff
@@ -9,26 +9,27 @@
 
 > A curated list of awesome things related to VuePress
 
-- [Resources](#resources)
-  - [Official Resources](#official-resources)
-  - [External Resources](#external-resources)
-  - [Tutorials](#tutorials)
-  - [Official Examples](#official-examples)
-  - [Examples](#examples)
-  - [Blog Posts](#blog-posts)
-- [Gallery](#gallery)
-- [Plugins](#plugins)
-  - [Official Plugins](#official-plugins)
-  - [Community Plugins](#community-plugins)
-- [Themes](#themes)
-  - [Official Themes](#official-themes)
-  - [Community Themes](#community-themes)
-- [Projects Using VuePress](#projects-using-vuepress)
-  - [Official Projects](#official-projects)
-  - [Blogging](#blogging)
-  - [Open Source](#open-source)
-  - [Enterprise Usage](#enterprise-usage)
-- [License](#license)
+1. [Awesome VuePress ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-vuepress-img-srchttpscdnrawgitcomsindresorhusawesomed7305f38d29fed78fa85652e3a63e154dd8e8829mediabadgesvg-altawesome)
```

Comment:
You can create a separate pull request for it

Reply:
Huh - wasn't mine!.  I'll close this one, re-fork the image, and re-PR.